README.md

```diff
@@ -1191,6 +1192,14 @@ messagingSession.addObserver(observer);
 messagingSession.start();
 ```
 
+### Amazon Voice Focus and Echo Reduction
+
+Amazon Voice Focus is a noise suppressor that uses machine learning to reduce unwanted background noise in your users’ microphone input. Unlike conventional noise suppressors, Amazon Voice Focus reduces fan noise, road noise, typing, rustling paper, lawnmowers, barking dogs, and other kinds of non-speech input, allowing your users to focus on the human voice. Amazon Voice Focus offers multiple complexity levels, allowing you to trade some quality to support a wider range of devices. The Amazon Chime SDK will by default automatically choose the right complexity level at runtime based on device performance.
+
+Echo reduction helps keep echoes—sounds from a user’s loudspeaker that get picked up by their microphone—from circulating back into meeting audio and bringing discussions to a standstill. The echo reduction feature is available in conjunction with the noise reduction provided by Amazon Voice Focus. Echo reduction is enabled at the meeting level when you call the CreateMeeting or CreateMeetingWithAttendees APIs. Enabling the feature this way allows others who join the meeting to enable echo reduction as desired.
```

Comment:
> Echo reduction helps keep echoes—sounds from a user’s 

Do you need the dash between echoes and sounds? `echoes-sounds` -> `echo`

Reply:
This is the official definition from the guide. The echo reduction model suppresses echo as well as sounds from the microphone input

Reply:
Well, just call it `echoes and sounds` then. It is confusing as an echo is also a sound so it is weird to use the dash there. Please update this in the guide.